econml/policy/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from .forest import PolicyTree, PolicyForest
+from ._drlearner import DRPolicyTree, DRPolicyForest
+
+__all__ = ["PolicyTree",
+           "PolicyForest",

```

Comment:
Isn't it a bit weird to expose these as both `econml.policy.PolicyTree` and also `econml.policy.forest.PolicyTree`?

Reply:
you mean to make forest into a private _forest?

Reply:
Yes, unless there are other things in there that we want to publicly expose one level down.

Reply:
done